{"name": "Проверка: если не передать query-параметр с треком, вернётся ошибка 400", "status": "passed", "start": 1710504336034, "stop": 1710504336603, "uuid": "617dfa80-517a-46b9-8413-2d81b92c3e2b", "historyId": "dcf6439dd6b010d7594a42cd2c9df220", "testCaseId": "dcf6439dd6b010d7594a42cd2c9df220", "fullName": "tests.tests_order.test_get_order_by_track.TestGetOrderByTrack#test_get_order_without_track_causes_400_error", "labels": [{"name": "parentSuite", "value": "tests.tests_order"}, {"name": "suite", "value": "test_get_order_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "Olyas-MacBook-Air.local"}, {"name": "thread", "value": "78521-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_order.test_get_order_by_track"}]}